docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -57,15 +57,21 @@ heartbeat.monitors:
   type: browser
   schedule: "@every 1m"
   source:
-    local: <3>
-      path: "/opt/todos" <4>
+    zip_url: <3>
+      url: "https://github.com/elastic/synthetics/archive/refs/heads/master.zip" <4>
+      folder: "examples/todos" <5>
+      username: "" <6>
+      password: "" 
 ----
 <1> Each `monitor` gets its own ID in the {uptime-app} and, therefore its own schedule entry.
 This allows tests to be run in parallel and analyzed separately.
 <2> In this example, a synthetic test is defined inline. This is a two-step script that first loads
 a homepage and then hovers over a product menu. See <<synthetics-syntax>> for more information.
-<3> In this example, our library of synthetic tests come from https://github.com/elastic/synthetics/tree/master/examples/todos[our todos example]. The path shown here corresponds to the path {heartbeat} will look at inside the docker container.
-<4> Note that this path is local to the docker container, not the host system. You can have the todos files anywhere you like on the host, and map them to `/opt/todos` in the container.
+<3> In this example, our library of synthetic tests is downloaded from the
+remote zip endpoint for https://github.com/elastic/synthetics/tree/master/examples/todos[our todos example]. 
+<4> Note that the url refers to the endpoint where the test project exists.
```

Comment:
Could we highlight these config keywords as code? Showing `url`, `folder`, `username`, `password` as code will add a subliminal element to this section that would make it easier for me to understand.

Then we could go back to lowercase "folder" and change "username" to be lower as well in item 6.

Reply:
Thanks Justin, Missed your review, will address it with a new PR.